ludwig/data/dataset/ray.py

```diff
@@ -101,6 +101,9 @@ def __len__(self):
     def size(self):
         return len(self)
 
+    def to_df(self):
+        return self.ds.to_dask()
```

Comment:
For this we should delegate to the `df_engine`, which could be Dask or Modin. Geoffrey has an implementation of this in his PR, which is still in review, but would make sense to adopt the same convention:

https://github.com/ludwig-ai/ludwig/pull/2040/files#diff-fec65d5ff8d68e96ebf11e5114a25e0ffa187739296f1e89d9949ae04cef0cc1R105

Reply:
Done.